### ISSUE TITLE
About Data

### ISSUE THREAD

**Issue by dgyf** [KangBoming/BulkFormer#7]
Dear Repository Maintainers,

I am writing to respectfully report an accessibility issue with one of the datasets within this repository. While reviewing the available data, I have been unable to successfully open “gtex_tissue_annot.h5ad”.

```
input:
adata = anndata.read_h5ad('gtex_tissue_annot.h5ad', backed="r")

output error:
OSError: Unable to synchronously open file (truncated file: eof = 1668284416, sblock->base_addr = 0, stored_eof = 3061505586)
```
Could you kindly provide guidance on whether there are specific access requirements I may have overlooked? Your assistance in resolving this matter would be greatly appreciated. Thank you for your time and consideration.

Best regards

**Comment by KangBoming** [comment]
Dear dgyf,

Thank you very much for reporting this issue.

The error message suggests that the uploaded `gtex_tissue_annot.h5ad` file may have been truncated or incompletely downloaded. We have checked the dataset and prepared a corrected version of the file.

You can download the corrected `gtex_tissue_annot.h5ad` file from the following Google Drive link:

https://drive.google.com/file/d/15NVZA8YLmQOsN_PGkjTZqzoQf3qSCpEg/view?usp=drive_link


We apologize for the inconvenience and appreciate you bringing this to our attention. We will also update the repository dataset link accordingly.

Best regards,  
Boming
